### PR TITLE
always create parameters.json as a post-build step

### PR DESCRIPTION
**Pull request by tjhei**
(sitting on top of #5709)

part of #5697

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -843,6 +843,16 @@ install(FILES ${CMAKE_BINARY_DIR}/aspect-gui
                 WORLD_READ WORLD_EXECUTE)
 endif()
 
+
+
+# target to create parameters.json after building aspect binary
+
+add_custom_command(TARGET ${TARGET} POST_BUILD
```
Comment (bangerth) [inline CMakeLists.txt:850]: Wow, `POST_BUILD`. Never seen that!

File: CMakeLists.txt [hunk 2]
```diff
@@ -843,6 +843,16 @@ install(FILES ${CMAKE_BINARY_DIR}/aspect-gui
...
+  COMMAND sh -c 'echo "" | ./aspect --output-json -- >parameters.json'
```
Comment (bangerth) [inline CMakeLists.txt:851]:
```suggestion
  COMMAND sh -c 'echo "" | ./aspect --output-json -- > parameters.json'
```

File: CMakeLists.txt [hunk 3]
```diff
@@ -843,6 +843,16 @@ install(FILES ${CMAKE_BINARY_DIR}/aspect-gui
                 WORLD_READ WORLD_EXECUTE)
 endif()
 
+
+
+# target to create parameters.json after building aspect binary
```
Comment (bangerth) [inline CMakeLists.txt:848]:
```suggestion
# A target to create parameters.json after building aspect binary.
```

**Review by bangerth — COMMENTED** [review]
Yes, looking good.

**Comment by tjhei** [comment]
fixed.

**Comment by bangerth** [comment]
Let me know once you've rebased post-#5709.

**Comment by tjhei** [comment]
done.

**Review by bangerth — APPROVED** [review]

**Pull request merged**